howdoi/howdoi.py

```diff
@@ -376,7 +377,7 @@ def _get_answer(args, link):
 
 
 def _get_links_with_cache(query):
-    cache_key = query + "-links"
+    cache_key = _get_cache_key(query + "-links")
```

Comment:
Here we can remove the `-links` because it was only there to do what `calling_func` does now.

Reply:
Ah right, I'll make that change